### ISSUE TITLE
[Feature Request] Setting to delete messages after X days

### ISSUE THREAD

**Issue by urmomsangus** [simplex-chat/simplex-chat#916]
Using on Android:

If I'm understanding the app correctly, all message attachments are stored on each device. My family and I use video and image attachments a lot which I worry will use a lot of internal storage.

It seems like there should be a way to autoclean the database or just have an auto delete setting.

Thanks for your help

**Comment by epoberezkin** [comment]
Are you asking to be able to automatically delete just local messages, is that correct?

**Comment by urmomsangus** [comment]
> Are you asking to be able to automatically delete just local messages, is that correct?

Yes, just locally for the user that sent the message and not the receivers.

This feature is about storage management and not privacy/security since I'm assuming large conversations will conversations will eventually take up large amounts of local storage -- please correct me if I'm wrong.

**Comment by jr-simplex** [comment]
@urmomsangus there's no auto-cleanup currently available, it may become a separate feature some time later; for now though you may want to periodically use Clear action on conversations that get most bloated to delete all messages and files associated with them. In chats list it's a swipe left on iOS / long press on Android, and also available as a button in the chat information sheet.
